### PR TITLE
Create LIT7067Qene.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

File: new/LIT7067Qene.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7067Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">">ንጽገብ፡ ለለዓመቱ፡ ወለተ፡ ኬፋ፡ እምደቂቅኪ፡ </title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Nǝṣgab lalla-ʿāmatu wallata kefā ʾǝm-daqiqǝki (Qǝne of the śǝllāse-type)</title>
+                <title xml:lang="en" corresp="#t1">Let us be saturated in each year from your offspring, daughter of Kephas (?) …</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7067Qene.xml:10]: Let us be saturated, year after year,

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT7067Qene.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7067Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">">ንጽገብ፡ ለለዓመቱ፡ ወለተ፡ ኬፋ፡ እምደቂቅኪ፡ </title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Nǝṣgab lalla-ʿāmatu wallata kefā ʾǝm-daqiqǝki (Qǝne of the śǝllāse-type)</title>
```
[the remark this answers is not in the document]
Comment (karljonaskarlsson) [reply new/LIT7067Qene.xml]: It's lalla- according to Leslau, but maybe there are other sources that say it should not be geminated

[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/LIT7067Qene.xml]: I think, we should be consistent with other Bm records. LIT3575Mashaf has lalla- and also zazza- in the title. I do not find any record, in which ለለ is transcribed with lala-.

[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT7067Qene.xml]: Indeed. It is a bit strange, but let it be. I do not think it has a good morphological explanation, in his dictionary there is a number of strange geminations.

**Pull request merged**